packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -112,6 +113,13 @@ class RefreshIndicator extends StatefulWidget {
   /// The progress indicator's background color. The current theme's
   /// [ThemeData.canvasColor] by default.
   final Color backgroundColor;
+  
+  /// A predicate that specifies whether a [ScrollNotification] should be
+  /// handled by this widget.
+  ///
+  /// By default, checks whether notification.depth == 0. Set it to something
```

Comment:
put backticks around the expression for better styling

Reply:
Done